packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -4,25 +4,31 @@
 
 #import "GoogleMapMarkerController.h"
 #import "FLTGoogleMapJSONConversions.h"
+#import "GMSMarker+Userdata.h"
 
 @interface FLTGoogleMapMarkerController ()
 
 @property(strong, nonatomic) GMSMarker *marker;
 @property(weak, nonatomic) GMSMapView *mapView;
 @property(assign, nonatomic, readwrite) BOOL consumeTapEvents;
+@property(strong, nonatomic) NSString *clusterManagerId;
+@property(strong, nonatomic) NSString *markerId;
```

Comment:
These need declaration comments (the existing ones missing comments are wrong too, but we shouldn't perpetuate that).